internal/types/metadata.go

```diff
@@ -38,4 +38,67 @@ type AppMetadata struct {
 	Description string
 	Namespace   string
 	Maintainers Maintainers
+	Parents     Parents
```

Comment:
https://github.com/docker/app/blob/master/specification/schemas/metadata_schema_v0.1.json should be updated as well (and the embedded schema should be updated too).